app/views/analytics/deal/_analytics_table.erb

```diff
@@ -0,0 +1,157 @@
+<div class="col-lg-12">
+  <div class="panel-group" id="accordion">
+    <div class="panel panel-primary">
+      <div class="panel-heading panel-heading-custom">
+        <h4 class="panel-title">
+          <a data-toggle="collapse" href="#collapseOne">
+            Active Deals
+            <span class="glyphicon glyphicon-triangle-bottom"></span>
+          </a>
+        </h4>
+      </div>
+      <div id="collapseOne" class="panel-collapse collapse in">
+        <div class="table-responsive">
+          <table class="table table-bordered table-hover">
+            <tr>
+              <th>Title</th>
+              <th style="text-align: center;">Conversion Rate<br>Wishlisters to Views</th>
+              <th style="text-align: center;">Conversion Rate<br>Unique Views to Redemptions</th>
+              <th style="text-align: center;">Percentage of users who redeem more than once</th>
+              <th style="text-align: center;">Average Time between same User Redemption</th>
+              <th colspan="1"></th>
+            </tr>
+
+            <% @deals.each do |deal| %>
+                <% if (deal.expiry_date.future?|| deal.expiry_date.today?) && deal.active %>
+                    <tr>
+                      <td><%= deal.title %></td>
+                      <% if @payment.add_on1 && deal.pushed %>
+                        <td style="text-align: center;"><%= DealAnalyticService.get_wishlist_to_views(deal.id) %>%</td>
+                      <% elsif @payment.add_on1 && !deal.pushed %>
+                          <td style="text-align: center;"><p id="untracable" data-toggle="tooltip" data-placement="top" title="Deal has not been pushed hence, data cannot be gathered">N/A</p></td>
+                      <% else %>
+                          <td style="text-align: center;"><p id="untracable" data-toggle="tooltip" data-placement="top" title="You have no purchased the push deal add_on hence, this feature is lock">N/A</p></td>
```

Comment:
"You have not purchased the Push Notification addon, hence you cannot view this"